gulpfile.js

```diff
@@ -103,8 +103,11 @@ gulp.task('copy:main.css', function () {
 
     return gulp.src(dirs.src + '/css/main.css')
                .pipe(plugins.header(banner))
-               .pipe(gulp.dest(dirs.dist + '/css'));
-
+                .pipe(plugins.autoprefixer({
+                    browsers: ['last 2 versions', 'ie >= 8'],
```

Comment:
@AllThingsSmitty: with that config we'll still loose the `(-o-min-device-pixel-ratio: 5/4)`. This conversation will last forever unless we define if we want to keep or drop support for old Opera desktop versions and/or Opera Mini/Mobile. @mathiasbynens: what's the consensus so far about this?

*FWIW my opinion is to drop support for those browsers, but these are two different topics.*